src/components/searchForm/searchForm.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { describe, expect, test } from 'vitest';
+import { screen, render, fireEvent } from '@testing-library/react';
+import SearchForm from './searchForm';
+import user from '@testing-library/user-event';
+import { vi } from 'vitest';
+
+describe('Search Form tests', () => {
+  test('check component initial search query correctly', async () => {
```

Comment:
1) Naming (need some verb)
2)why async? (it is required if you are using await)
3)why user.setup()?  (it is required if you are user interactions)